app/views/shared/_location_search_tag.html.haml

```diff
@@ -1,7 +1,7 @@
 .govuk-form-group.js-location-finder#location-search
   = label_tag 'location', t('jobs.filters.location'), class: "govuk-label #{extra_label_class}"
   .js-location-finder__input-container#js-location-finder__input-container
-    = search_field_tag :location, @vacancies_search&.location, class: 'govuk-input js-location-finder__input app-site-search__input app-site-search__input--default', placeholder: t('jobs.filters.location_hint'), autocomplete: 'off',  "aria-expanded": "true", "aria-owns": "location__listbox", "aria-autocomplete": "list", role: "combobox"
+    = search_field_tag :location, @vacancies_search&.location, class: 'govuk-input js-location-finder__input app-site-search__input app-site-search__input--default', placeholder: t('jobs.filters.location_hint'), autocomplete: 'off',  "aria-expanded": "true", "aria-owns": "location__listbox", "aria-autocomplete": "list", role: "combobox", 'data-coordinates': @vacancies_search&.coordinates
```

Comment:
`data: { coordinates: @vacancies_search&.coordinates }` should do the job too if you prefer